examples/dom/flexbox.cpp

```diff
@@ -0,0 +1,38 @@
+#include <stddef.h>                // for size_t
```

Comment:
I think this example doesn't compile yet. It's not included in the tests CMakeLists but if it's unused at the moment, I think it shouldn't be in the master examples.